src/Turbo/assets/src/turbo_stream_controller.ts

```diff
@@ -40,7 +42,7 @@ export default class extends Controller {
 
     connect() {
         if (this.url) {
-            this.es = new EventSource(this.url);
+            this.es = new EventSource(this.url, this.eventSourceOptionsValue);
```

Comment:
What would happen if evenSourceOptionsValue is not set ?

Reply:
Object empty by default via stimulus.